frontend/src/pages/events/index.js

```diff
@@ -7,29 +7,42 @@ import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEve
 import { StaticImage } from "gatsby-plugin-image";
```

Comment:
add a comment with screenshots of desktop and mobile views so we can review more easily